parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -157,7 +157,7 @@ $global:AzureCNIConfDir = [Io.path]::Combine("$global:AzureCNIDir", "netconf")
 # $global:NetworkPolicy = "{{GetParameter "networkPolicy"}}" # BUG: unused
 $global:NetworkPlugin = "{{GetParameter "networkPlugin"}}"
 $global:VNetCNIPluginsURL = "{{GetParameter "vnetCniWindowsPluginsURL"}}"
-$global:IsDualStackEnabled = {{if IsIPv6DualStackFeatureEnabled}}$true{{else}}$false{{end}}
+$global:IsDualStackEnabled = {{if or IsIPv6DualStackFeatureEnabled IsAzureCNIOverlayDualStackFeatureEnabled}}$true{{else}}$false{{end}}

```

Comment:
Do we need any change for DualStack with WinDSR enabled?
Current `Set-AzureCNIConfig` in staging/cse/windows/azurecnifunc.ps1 only handles DualStack when WinDSR is disabled.

Reply:
Do you know if there's any reason they need to be mutually exclusive? Can the dualstack section in `Set-AzureCNIConfig` run regardless of if DSR is enabled?

Reply:
We only have one customer with WinDSR disabled. @allyford is helping on this to enable WinDSR in their clusters.
I think that we can remove all code for WinDSR disabled.